fixtures/generate.py

```diff
@@ -0,0 +1,75 @@
+# For instructions on using this script, please see the README.
+
+from tuf import repository_tool as rt
+import os
+import shutil
+
+from os import path
+from unittest import mock
+
+# This file largely derives from the TUF tutorial:
+# https://github.com/theupdateframework/tuf/blob/develop/docs/TUTORIAL.md
+
+def import_keypair(name):
+    dir = path.join(os.getcwd(), 'keys')
+    private_path = path.join(dir, name)
+    public_path = private_path + '.pub'
+
+    # Load the keys into TUF.
+    public = rt.import_ed25519_publickey_from_file(public_path)
+    private = rt.import_ed25519_privatekey_from_file(private_path, password='pw')
+
+    return (public, private)
+
+def add_target(repository, filename, signing_target=None):
+    if signing_target is None:
+        repository.targets.add_targets([filename])
+    else:
+        repository.targets(signing_target).add_target(filename)
+        repository.mark_dirty([signing_target])
+
+    repository.mark_dirty(['snapshot', 'targets', 'timestamp'])
```

Comment:
If we got rid of `add_target` function then I think we can just call this after both targets have been added.

Reply:
See above.